src/libAtomVM/term.h

```diff
@@ -51,19 +52,26 @@
 #define TERM_BOXED_REFC_BINARY 0x20
 #define TERM_BOXED_HEAP_BINARY 0x24
 
-#define TERM_BOXED_REFC_BINARY_SIZE 3
-#define TERM_BOXED_BIN_MATCH_STATE_SIZE 4
+#define TERM_BOXED_REFC_BINARY_SIZE 6
+#define TERM_BOXED_BIN_MATCH_STATE_SIZE 6
 
 #define BINARY_HEADER_SIZE 2
 #define FUNCTION_REFERENCE_SIZE 4
 #define BOXED_INT_SIZE (BOXED_TERMS_REQUIRED_FOR_INT + 1)
 #define BOXED_INT64_SIZE (BOXED_TERMS_REQUIRED_FOR_INT64 + 1)
 #define FLOAT_SIZE (sizeof(float_term_t) / sizeof(term) + 1)
+#define REFC_BINARY_CONS_OFFET  4
+#define MATCH_BINARY_CONS_OFFET  4
 
 #define TERM_DEBUG_ASSERT(...)
 
 #define TERM_FROM_ATOM_INDEX(atom_index) ((atom_index << 6) | 0xB)
 
+enum RefcBinaryFlags
+{
+    refcBinaryIsConst = 0x01
```

Comment:
RefcBinaryIsConst since it is an enum value

Reply:
Fixed.